gatsby-config.js

```diff
@@ -61,6 +61,10 @@ module.exports = {
             title: "Integration with Amazon EventBridge",
             path: "guides/amazon_eventbridge/index.md",
           },
+          {
+            title: 'InDesign Cloud Services Events',
+            path: 'guides/using/idcs/idcs-events-data-stream-setup.md'
+          }
```

Comment:
This is configured at the wrong place. It should be after https://github.com/AdobeDocs/adobe-io-events/blob/main/gatsby-config.js#L147